coderd/notifications/notifications_test.go

```diff
@@ -517,6 +517,73 @@ func TestInvalidConfig(t *testing.T) {
 	require.ErrorIs(t, err, notifications.ErrInvalidDispatchTimeout)
 }
 
+func TestNotifierPaused(t *testing.T) {
+	t.Parallel()
+
+	// setup
+	ctx := testutil.Context(t, testutil.WaitLong)
```

Comment:
Nit: btw now we have `setupInMemory` since https://github.com/coder/coder/pull/13863 was merged.

Reply:
Fixed 👍